Source/Krypton Toolkit/Shared/Krypton.Toolkit.Suite.Extended.Tools/Classes/General/MissingFrameWorkAPIs.cs

```diff
@@ -39,5 +41,17 @@ public static bool IsNullOrWhiteSpace(string value)
             return true;
 
         }
+
+        /// <summary>Splits the array.</summary>
+        /// <param name="inputValue">The input value.</param>
+        /// <param name="delimiter">The delimiter.</param>
+        /// <param name="limit">The limit.</param>
+        /// <returns>The values.</returns>
+        public static string[] SplitArray(string inputValue, string delimiter, int limit = -1)
+        {
+            string[] output = Strings.Split(inputValue, delimiter, limit, CompareMethod.Text);
```

Comment:
This is surely not needed:
What's wrong with inputValue.Split( delimiter ) ?

Reply:
Could do as the line is `string[] RGBColorArray = Microsoft.VisualBasic.Strings.Split(InputColor, ",", -1, CompareMethod.Text);`